src/emqx_ctl.erl

```diff
@@ -153,11 +153,11 @@ usage(CmdParams, Desc) ->
 
 -spec(format(io:format()) -> string()).
 format(Msg) ->
-    lists:flatten(io_lib:format("~s", [Msg])).
+    replace(lists:flatten(io_lib:format("~s", [Msg]))).
```

Comment:
wait. change line 144 to `io:format("~s", [format(Format, Args)])` ?

Reply:
`~s` cannot solve the problem that an error is reported when there is a "~" in the printed content
```
1> io:format("~s", "~").
** exception error: bad argument
     in function  io:format/3
        called as io:format(<0.66.0>,"~s","~")
2> io:format("~").
** exception error: bad argument
     in function  io:format/3
        called as io:format(<0.66.0>,"~",[])
3> io:format("~~").
~ok
```